src/TestEnvironment.Docker.Containers.Mongo/IMongoContainer.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace TestEnvironment.Docker.Containers.Mongo
+{
+    public interface IMongoContainer
+    {
+        string GetConnectionString();
```

Comment:
I think such interface should be extracted to the main project and be named like 'IDatabaseContainer'

Reply:
And what benefits it will bring? In my case, it's needed to be sure that Mongo connection string will be returned in MongoCleaner. In the case of introducing shared `IDatabaseContainer` there is no guarantee that the proper container will be passed into the ` public Task Cleanup(Container container, CancellationToken token = default)` method.
I suggest moving such discussion out of such PR, WDYT?

Reply:
Agree, placing `IDatabaseContainer` into the core project is not very obvious decision.

Reply:
Sure, lets discuss later